components/as-ohttp-client/Cargo.toml

```diff
@@ -25,3 +25,7 @@ features = ["client", "server", "app-svc", "external-sqlite"]
 
 [build-dependencies]
 uniffi = { workspace = true, features=["build"]}
+
+# Temp hack for uniffi 0.28 generated code, will not be necessary after next uniffi update.
+[lints.clippy]
+empty-line-after-doc-comments = "allow"
```

Comment:
This is very unfortunate :( We could put this in the top-level Cargo.toml, until we get to m-c, where we will be unable to use workspace attributes for almost all of these crates.

Reply:
I think this is totally OK for now—as you said in the comment, it's temporary!